### PR TITLE
Update openvpn-gui-res-zh-hans.rc

### PR DESCRIPTION
**Pull request by justwho**
Update openvpn-gui-res-zh-hans.rc

### CODE REVIEW

**Comment by lstipakov** [comment]
@selvanair I asked my colleague who speaks Mandarin to check, he said it looks good.

**Review by lstipakov — APPROVED** [review]

File: res/openvpn-gui-res-zh-hans.rc [hunk 1]
```diff
@@ -104,7 +104,7 @@ BEGIN
     LTEXT "", ID_TXT_IP, 20, 160, 300, 10
     PUSHBUTTON "断开连接", ID_DISCONNECT, 50, 190, 50, 14
     PUSHBUTTON "重新连接", ID_RESTART, 150, 190, 50, 14
-    PUSHBUTTON "De&tach", ID_DETACH, 180, 190, 50, 14, WS_DISABLED
+    PUSHBUTTON "断开", ID_DETACH, 180, 190, 50, 14, WS_DISABLED
```
Comment (huiwenliu95) [inline res/openvpn-gui-res-zh-hans.rc:107]: Maybe it's better to use "分离"/"脱离" to distinguish from "断开连接".

Comment (selvanair) [reply res/openvpn-gui-res-zh-hans.rc:107]: @justwho Any thoughts on this? Here "Detach" refers to detaching the GUI from the VPN process, leaving the connection state  unchanged. "Disconnect" means to stop the VPN connection. Is it possible to use distinct words for the two as they would appear side by side?

Comment (selvanair) [reply res/openvpn-gui-res-zh-hans.rc:107]: > @selvanair ok. @huiwenliu95 is right.

Could you please choose one of the suggested alternatives and add a commit to this? I can squash them while merging.

Comment (justwho) [reply res/openvpn-gui-res-zh-hans.rc:107]: Yes.I agree to change to“分离”,Thanks!

**Pull request merged**